doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -171,23 +171,87 @@ is typically done in the REST interface)
 (placeholder to describe roles like context provider, context producer,
 context broker, context registry, context consumer)
 
-## JSON entity representation
+## JSON Entity Representation
 
-An entity is represented by a a JSON object which contain the following properties:
+An entity is represented by a JSON object with the following syntax:
 
-* `id` for the entity ID, represented by a JSON string. If the entity has an `id` attribute itself it will be ignored.
-* `type` for the entity type, represented by a JSON string. If the entity has a `type` attribute itself it will be ignored.
-  If the entity doesn't have a type, then this property is not included.
-* A collection of properties, one per entity attribute. Properties representing attributes with no metadata
-  or type are rendered as regular JSON values. Properties representing attributes which specify a type or include
-  metadata are rendered as JSON objects with the following properties:
-  * `value`: for the attribute value, rendered as a regular JSON value (string, number or boolean), object or array.
-  * `type`: a JSON string which represents the user-defined NGSI attribute type.
-  * A collection of properties one per metadata field associated to the attribute value. Metadata properties follow
-    the same representation rules that applies to properties which represent attributes (except that a metadata
-    cannot have metadata properties).
+* The entity id is specified by the object's `id` property, which value will be a string containing the entity id.
 
-### Special attribute types
+* The entity type is specified by the object's `type` property, which value will be a string containing the entity's type name.
+
+* Entity attributes are specified by additional properties, which name is the `name` of the attribute and which representation
+  is described in the JSON Attribute Representation section below. Obviously, `id` and `type` are not allowed as attribute names.
+
+An example of this syntax in shown below:
+
+
+```
+{
+  "id": "entityID",
+  "type": "entityType",
+  "attr_1": <val_1>,
+  "attr_2": <val_2>,
+  ...
+  "attr_N": <val_N>
+}
+```
+
+
+## JSON Attribute Representation
+
+An attribute is represented by a JSON object with the following syntax:
+
+* The attribute value is specified by the `value` property, which value can be any JSON datatype.
+
+* The attribute NGSI type is specified by the `type` property, which value will be a string containing the attribute NGSI type.
```

Comment:
a string containing the NGSI type name. 

(it is understood that it refers to the attribute)